go.mod

```diff
@@ -5,7 +5,7 @@ go 1.15
 require (
 	github.com/Masterminds/semver v1.5.0
 	github.com/fatih/color v1.9.0
-	github.com/fi-ts/cloud-go v0.8.0
+	github.com/fi-ts/cloud-go v0.8.1
 	github.com/gardener/gardener v1.8.2
 	github.com/go-openapi/runtime v0.19.21
```

Comment:
can you please update metal-lib to v0.6.0 an security to v0.4.0 as well

Reply:
Done.